c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -232,15 +235,24 @@ MfxC2DecoderComponent::MfxC2DecoderComponent(const C2String name, const CreateCo
         }
         case DECODER_VP8: {
             supported_profiles = {
+#if MFX_ANDROID_VERSION <= MFX_R
                 (C2Config::profile_t)PROFILE_VP8_0,
```

Comment:
Can you help to explain why the C2Config::profile_t is not needed for Android S?

Reply:
Have explained here where VP8 profile defined for Android R. :)
+// Android S declared VP8 profile
+#if MFX_ANDROID_VERSION <= MFX_R
 enum VP8_PROFILE {
     PROFILE_VP8_0 = C2_PROFILE_LEVEL_VENDOR_START,
 };
+#endif